doc/website/advanced/iceoryx_utils.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: Safe building blocks
 ---
-{! ./../iceoryx_utils/README.md !}
+{! ../iceoryx/iceoryx_utils/README.md !}
```

Comment:
Let's say I do not believe you that the path is now correct. How can I verify the correctness of those relative paths as a reviewer?
Why is it relative at all? Couldn't we use absolute paths here?

Reply:
See [answer](https://github.com/eclipse-iceoryx/iceoryx/pull/746#discussion_r615822093) in the original pull request. The sanity check should be sufficient?